### ISSUE TITLE
Laravel 8 Support

### ISSUE THREAD

**Issue by vandadsoft** [cyrildewit/eloquent-viewable#233]
Laravel 8 Update Support

**Comment by prof-usman** [comment]
yes please

**Comment by bigint** [comment]
Yes please add support for Laravel 8

**Comment by MB116** [comment]
+1

**Comment by prof-usman** [comment]
still waiting

**Comment by DubStepMad** [comment]
Hopefully soon

**Comment by oladejihenry** [comment]
Still waiting

**Comment by bigint** [comment]
Already a PR is there #237

**Comment by reliq** [comment]
You can already use the proposed laravel 8 compatible version with:

```
composer require cyrildewit/eloquent-viewable:dev-laravel-8-support
```

**Comment by oladejihenry** [comment]
@reliq i keep getting this error

```
[InvalidArgumentException]                                                                       
  Could not find package cyrildewit/eloquent-viewable in a version matching dev-laravel-8-support
```

**Comment by reliq** [comment]
@oladejihenry sorry I forgot to mention you should also add `https://github.com/reliq/eloquent-viewable` to the repositories section of your `composer.json` like so:

```
"require": {
        // ...
        "cyrildewit/eloquent-viewable": "dev-laravel-8-support",
        // ...
},
"repositories": [
        {
            "url": "https://github.com/reliq/eloquent-viewable",
            "type": "vcs"
        }
],
```

**Comment by cyrildewit** [comment]
Hey everyone,

Thank you for your patience. I've released `v5.2.1` with support for Laravel 8.
